docs/client/javascript-mono/MigrationFromOldJsClient.mdx

```diff
@@ -244,9 +244,13 @@ StatsigLocalStorage.disabled = false;
 </Tabs>
 
 
-## getStableID
+## stableID and getStableID
 `statsig.getStableID()` no longer exists. You can get the stableID like this: `myStatsigClient.getContext().stableID`
 
+The key for the stableID in local storage has changed. It is now a function of the SDK key used to initialize the sdk, which enables multiple instances of the SDK to coexist on a single page and not overlap with cached data.  If you wish to keep the existing stableID, you should access it in local storage from the old key `STATSIG_LOCAL_STORAGE_STABLE_ID`, and set it in `customIDs` to override the stableID.
+
+Similarly, if you rely on cached values from a previous SDK integration and want those to persist, you should...
```

Comment:
@daniel-statsig what should we suggest here?   Is there a migration path?  It would be quite difficult to read the old cache, parse out the user, and then bootstrap, but I cant think fo any other way